cmd/packages.pony

```diff
@@ -113,8 +118,9 @@ class val Package is Comparable[Package box]
 type CPU is AMD64
 primitive AMD64
 
-type OS is (Linux | Darwin)
+type OS is (Linux | Darwin | FreeBSD)
```

Comment:
We can define the type `OS` as follows to allow us to use `h.assert_eq[OS]` in the unit tests:
```pony
type OS is ((Linux | Darwin | FreeBSD) & _OS val)
interface val _OS is (Equatable[_OS] & Stringable)
primitive Linux is _OS
  fun string(): String iso^ => "linux".string()
...
```